source/config/Config.cpp

```diff
@@ -252,6 +252,13 @@ bool PlainConfig::LoadFromEnvironment()
         }
     }
 
+    const char *lockFilePathStr = std::getenv("LOCK_FILE_PATH");
+    if (lockFilePathStr)
+    {
+        LOGM_DEBUG(Config::TAG, "Set LOCK_FILE_PATH=%s", lockFilePathStr);
```

Comment:
Please validate the path ad save the `Expanded path` over here instead of the value received from env variable. Also `Sanitize` the expanded path passed in LOG statement.